src/dcdc.cpp

```diff
@@ -49,7 +49,7 @@ void dcdc_init(Dcdc *dcdc)
 // returns if output power should be increased (1), decreased (-1) or switched off (0)
 int _dcdc_output_control(Dcdc *dcdc, DcBus *out, DcBus *in)
 {
-    float dcdc_power_new = out->voltage * out->current;
+    float dcdc_power_new = dcdc->ls_voltage * dcdc->ls_current;
```

Comment:
I think it must be `fabs(dcdc->ls_voltage * dcdc->ls_current)`, as otherwise the MPPT won't work for negative currents in boost mode.

Reply:
Yes, you're right. Fixed that. 
However, it would be best to know here which direction current is supposed to flow.